scripts/configure.py

```diff
@@ -837,42 +855,46 @@ def _update_web(env: dict, restart_updater: bool, progress) -> List[Task]:
-  # This fixes some potential update issues caused by migrating install to a different directory
-  # (using the web updated the install dir used to be amplipi-dev2 and is now amplipi-dev)
-  _copy_old_config(env['base_dir'])
+
   tasks = []
-  # stop amplipi before reconfiguring authbind
-  tasks += print_progress(_stop_service('amplipi'))
+
+  if not env['is_ci']:
+    # try to copy the old config into the potentially new directory
+    # This fixes some potential update issues caused by migrating install to a different directory
+    # (using the web updated the install dir used to be amplipi-dev2 and is now amplipi-dev)
+    _copy_old_config(env['base_dir'])
+
+    # stop amplipi before reconfiguring authbind
+    tasks += print_progress(_stop_service('amplipi'))
+
   # bringup amplipi and updater separately
   tasks += print_progress(_configure_authbind())
-  tasks += print_progress(_create_service('amplipi',
-                          _web_service(env['base_dir'])))
-  tasks += print_progress(_start_service('amplipi',
-                          test_url='http://0.0.0.0'))
-  if not tasks[-1].success:
-    return tasks
-  tasks += print_progress([_check_version('http://0.0.0.0/api')])
-  tasks += print_progress(_create_service('amplipi-updater',
-                          _update_service(env['base_dir'])))
-  if restart_updater:
+  tasks += print_progress(_create_service('amplipi', _web_service(env['base_dir']), env))
+  tasks += print_progress(_enable_service('amplipi'))
+  if not env['is_ci']:
+    tasks += print_progress(_start_service('amplipi', test_url='http://0.0.0.0'))
+    if not tasks[-1].success:
+      return tasks
+    tasks += print_progress([_check_version('http://0.0.0.0/api')])
+
+  tasks += print_progress(_create_service('amplipi-updater', _update_service(env['base_dir']), env))
+  tasks += print_progress(_enable_service('amplipi-updater'))
+  if restart_updater and not env['is_ci']:
     tasks += print_progress(_stop_service('amplipi-updater'))
-    tasks += print_progress(_start_service('amplipi-updater',
-                            test_url='http://0.0.0.0:5001/update'))
-  else:
+    tasks += print_progress(_start_service('amplipi-updater', test_url='http://0.0.0.0:5001/update'))
+  elif not restart_updater and not env['is_ci']:
     # start a second updater service and check if it serves a url
     # this allow us to verify the update the updater probably works
-    tasks += print_progress(_create_service('amplipi-updater-test',
-                            _update_service(env['base_dir'], port=5002)))
-    tasks += print_progress(_start_service('amplipi-updater-test',
-                            test_url='http://0.0.0.0:5002/update'))
+    tasks += print_progress(_create_service('amplipi-updater-test', _update_service(env['base_dir'], port=5002), env))
+    tasks += print_progress(_start_service('amplipi-updater-test', test_url='http://0.0.0.0:5002/update'))
```

Comment:
I think this reads a little better if the whole thing is wrapped in if not env['ci']

Reply:
oh nice yea totally